api/android/build-android-lib.sh

```diff
@@ -62,6 +62,7 @@ enable_tflite="yes"
 
 # Set tensorflow-lite version (available: 1.9.0 / 1.13.1 / 1.15.2 / 2.3.0)
 tf_lite_ver="1.13.1"
+tf_lite_vers_support="1.9.0 1.13.1 1.15.2 2.3.0"
```

Comment:
Please add a help message (e.g., --help) how to use options.
Especially, ```--enable_tflite``` is not anymore trivial.

Reply:
@myungjoo Great idea!